src/config/kernel.md

```diff
@@ -18,7 +18,8 @@ modules.
 
 When updating the kernel, old versions are left behind in case it is necessary
 to roll back to an older version. Over time, old kernel versions can accumulate,
-consuming disk space and increasing the time taken by dkms module updates. Thus,
+consuming disk space and increasing the time taken by dkms module updates. If
+the boot partition fills up with old kernels, it can cause a kernel panic. Thus,
```

Comment:
Suggest: "Furthermore, if `/boot` is a separate partition and fills up with old kernels, it can cause a kernel panic."

Reply:
Amended!